components/hook-esm/node/index.test.mjs

```diff
@@ -30,37 +31,43 @@ assertDeepEqual(
       url: "file:///base",
     },
     async () => {
-      global.APPMAPuuid = {
+      globalThis.APPMAPuuid = {
         getFreshTab: () => 123,
       };
       // transformSource //
       {
         const { source } = await APPMAP_ESM_HOOK.transformSource(
           "123;",
           { url: "file:///foo", format: "module" },
-          async (content, context, next) => ({ source: content }),
+          async (content, _context, _next) => ({ source: content }),
```

Comment:
Same thing here. Without an `await` there's no need to mark a function as `async`.

Reply:
cf above